### ISSUE TITLE
Where are the Emacs Lisp communities and enthusiasts?

### ISSUE THREAD

**Issue by kytrinyx** [exercism/elisp#86]
As we move towards the launch of the new version of Exercism we are going to be ramping up on actively recruiting people to help provide feedback.

Our goal is to get to 100%: everyone who submits a solution and wants feedback should get feedback. Good feedback. You can read more about this aspect of the new site here: http://mentoring.exercism.io/

To do this, we're going to need a lot more information about where we can find language enthusiasts.

* Is Emacs Lisp supported by one or more large organizations?
* Does Emacs Lisp have an official community manager?
* Do you know of specific communities (online or offline) that are enthusiastic about Emacs Lisp? (Chat communities, forums, meetups, student clubs, etc)
* Are there popular conferences for Emacs Lisp? (If so, what are some examples?)
* Are there any organizations who are targeted specifically at getting certain subgroups or demographics interested in Emacs Lisp? (e.g. kids, teenagers, career changers, people belonging to various groups that are typically underrepresented in tech?)
* Are there specific groups or programs dedicated to mentoring people in Emacs Lisp?
* Are there popular newsletters for Emacs Lisp?
* Is Emacs Lisp taught at programming bootcamps? (If so, what are some examples?)
* Is Emacs Lisp taught at universities? (If so, what are some examples?)

In other words: where do people care a lot and/or know a lot about Emacs Lisp?

This is part of the project being tracked in https://github.com/exercism/meta/issues/103

**Comment by benreyn** [comment]
> Is Emacs Lisp supported by one or more large organizations?

To some extent the [Free Software Foundation](https://www.fsf.org/) supports development of GNU(https://www.gnu.org/) projects including emacs as well as [guile](https://www.gnu.org/software/guile/) which happens to be the primary implementation of scheme recommended for use in the [scheme track](https://github.com/exercism/scheme) There is also [frontmacs](https://github.com/thefrontside/frontmacs) (one of many emacs configuration distributions) is supported by frontside.io to some degree. 

> Does Emacs Lisp have an official community manager?

Official...? 🤷‍♂️
 
> Do you know of specific communities (online or offline) that are enthusiastic about Emacs Lisp? (Chat communities, forums, meetups, student clubs, etc)

* [New Tork Emacs meetup](http://emacsnyc.org/)
* [Boston Emacs Meetup](https://www.meetup.com/Boston-Emacs-Meetup/)
* [Spacemacs](https://github.com/syl20bnr/spacemacs) Users and developers ([Gitter Chat](https://gitter.im/syl20bnr/spacemacs))
* [doom-emacs](https://github.com/hlissner/doom-emacs) Users and developers ([Discord Chat](https://discord.gg/bcZ6P3y))
* [Prelude](https://github.com/bbatsov/prelude) Users and developers ([Gitter Chat](https://gitter.im/bbatsov/prelude))
* [Remacs](https://github.com/Wilfred/remacs) Developers ([Gitter Chat](https://gitter.im/remacs-discuss/Lobby)) Who, coincidentally, are also Rust enthusiasts.

> Are there popular conferences for Emacs Lisp? (If so, what are some examples?)

[Once](https://emacsconf.github.io/emacsconf2015/) back in 2015, but never repeated as far as I know. 😞 

> Are there any organizations who are targeted specifically at getting certain subgroups or demographics interested in Emacs Lisp? (e.g. kids, teenagers, career changers, people belonging to various groups that are typically underrepresented in tech?)

Nope 😞
 
> Are there specific groups or programs dedicated to mentoring people in Emacs Lisp?

Nope 😞 

> Are there popular newsletters for Emacs Lisp?

I _think_ there's a mailing list for the developers of emacs itself?

> Is Emacs Lisp taught at programming bootcamps? (If so, what are some examples?)

Nope 😞 

> Is Emacs Lisp taught at universities? (If so, what are some examples?)

Nope 😞

**Comment by kytrinyx** [comment]
I've got what I need for now; closing this out. (Should you come across any other good sources, feel free to add them here, I'll be coming back to this). 🌸  Thanks!

**Comment by brandelune** [comment]
The issue is closed but I have some additional information:

> Is Emacs Lisp supported by one or more large organizations?

As written above, the FSF that directly sponsors the GNU project where emacs has iconic status.

> Does Emacs Lisp have an official community manager?

Emacs Lisp and Emacs are to be considered the same project, as such the current Emacs maintainer is John Wiegley and so can be considered the official Emacs Lisp community manager.

> Do you know of specific communities (online or offline) that are enthusiastic about Emacs Lisp? (Chat communities, forums, meetups, student clubs, etc)

The Emacs development list and the Emacs help list (that's for official GNU lists).
    https://lists.gnu.org/mailman/listinfo/help-gnu-emacs
    https://lists.gnu.org/mailman/listinfo/emacs-devel
The Emacs subreddit.
    https://www.reddit.com/r/emacs/
And the usenet groups.

> Are there popular conferences for Emacs Lisp? (If so, what are some examples?)

As written above, no regular conference, sometimes meetings when developers gather.

> Are there any organizations who are targeted specifically at getting certain subgroups or demographics interested in Emacs Lisp? (e.g. kids, teenagers, career changers, people belonging to various groups that are typically underrepresented in tech?)

I don't think so.

> Are there specific groups or programs dedicated to mentoring people in Emacs Lisp?

Since Emacs Lisp is primarily used for Emacs related development, the emacs-devel and help-gnu-emacs list can be considered as such groups. Any would-be Emacs Lisp developer who has code can ask for advice and senior developers will do code reviews as well as point at useful ressources.

> Are there popular newsletters for Emacs Lisp?

planet.emacsen is the place where blogs that are considered relevant are gathered:
    http://www.planet.emacsen.org

Also, Sacha Chua has a weekly newsletter that gives links to the most comprehensive list of Emacs/Emacs Lisp related items she found on the web:
    http://sachachua.com/blog/category/geek/emacs/

> Is Emacs Lisp taught at programming bootcamps? (If so, what are some examples?)
> Is Emacs Lisp taught at universities? (If so, what are some examples?)

I just sent a message to help.gnu.emacs. If there are any replies I'll add the information here. Also, Emacs is considered the IDE of choice for Lisp development. So basically any university that has Lisp classes will probably teach some basic Emacs use and Emacs Lisp.

> In other words: where do people care a lot and/or know a lot about Emacs Lisp?

*Definitely* the emacs-devel and help.gnu.emacs lists.
